cartographer/transform/proto/transform_interpolation_buffer.proto

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The Cartographer Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto2";
+
+package cartographer.transform.proto;
+
+import "cartographer/transform/proto/transform.proto";
+
+message TransformInterpolationBuffer {
+  message StampedTransform {
```

Comment:
There is already a NodeData proto in the sensor package, but I think it semantically makes sense to introduce a stamped transform proto for serializing the interpolation buffer.